pkg/logical-cluster-manager/cluster-manager-controller.go

```diff
@@ -0,0 +1,244 @@
+package clustermanager
```

Comment:
You will want the usual boilerplate in these files, as soon as they appear in public.